services/wallet/api.go

```diff
@@ -445,6 +445,25 @@ func (api *API) GetSuggestedRoutes(
 		disabledToChaindIDs, preferedChainIDs, gasFeeMode, fromLockedAmount)
 }
 
+func (api *API) GetSuggestedRoutesV2(
+	ctx context.Context,
+	sendType SendType,
+	addrFrom common.Address,
+	addrTo common.Address,
```

Comment:
I see a lot of these parameters already exists so I don't know easy a change this is.
However, I find parameter/variable names that drop letters to be confusing. 
Sure we save a few characters but it's generally just better to use full words so the developer doesn't have to guess what the meaning is (not everyone is a native speaker) and on the otherside you almost have to guess what letters have been dropped as this is often an inconsistent and random choice. 
Overall it also leaves the codebase cleaner imo and that serves a lot in the long run. ⚔️